pkgs/by-name/sc/scripthaus/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "scripthaus";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "scripthaus-dev";
+    repo = "scripthaus";
+    rev = "v${version}";
+    hash = "sha256-ZWOSLkqjauONa+fKkagpUgWB4k+l1mzEEiC0RAMUmo0=";
+  };
+
+  vendorHash = "sha256-GUZNPLBgqN1zBzCcPl7TB9/4/Yk4e7K6I20nVaM6ank=";
+
+  CGO_ENABLED = 1;
+
+  ldflags = [ "-s" "-w" ];
+
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/${pname}
```

```suggestion
    mv $out/bin/cmd $out/bin/scripthaus
```
to match `meta.mainProgram`

---

i'm not golang expert (even golang programmer, i'm c# 🙂). On their [github](https://github.com/scripthaus-dev/scripthaus) they say it should be build using `CGO_ENABLED=1 go build -o scripthaus cmd/main.go`. I've tried other ways but without success, so i thought "why not KISS?". 

Do you have any idea to improve this?